presto-main/src/main/java/io/prestosql/server/ClusterStatsResource.java

```diff
@@ -56,12 +56,16 @@ public ClusterStats getClusterStats()
         long runningQueries = 0;
         long blockedQueries = 0;
         long queuedQueries = 0;
-
-        long activeNodes = nodeManager.getNodes(NodeState.ACTIVE).size();
-        if (!isIncludeCoordinator) {
-            activeNodes -= 1;
+        long activeNodes = nodeManager.getNodes(NodeState.ACTIVE).stream()
```

Comment:
If we change this to:
```
long activeWorkers = nodeManager.getNodes(NodeState.ACTIVE).stream()
                .filter(node -> isIncludeCoordinator || !node.isCoordinator())
                .count();
```
It will ignore coordinators in this count unless, `isIncludeCoordinator` is set, so if you have multiple coordinator and are including them, you get the right count.